server/repository/src/db_diesel/mod.rs

```diff
@@ -312,7 +312,16 @@ impl From<DieselError> for RepositoryError {
                 Error::as_db_error("DIESEL_INVALID_C_STRING", extra)
             }
             DieselError::DatabaseError(err, extra) => {
-                let extra = format!("{:?}", extra);
+                let extra = format!(
```

Comment:
Ok so the {:?} on extra was showing with lots of `\\\\\` making it hard to read, this change seems to show the error much better with just one `\`